rdrf/rdrf/views/dashboard_view.py

```diff
@@ -125,12 +125,23 @@ def _get_cde_data(self, cfg, form, section, cde):
         if not data:
             return None
 
-        form_value = self.patient.get_form_value(self.registry.code,
-                                                 form.name,
-                                                 section.code,
-                                                 cde.code,
-                                                 multisection=section.allow_multiple,
-                                                 clinical_data=data)
+        try:
+            form_value = self.patient.get_form_value(self.registry.code,
+                                                     form.name,
+                                                     section.code,
+                                                     cde.code,
+                                                     multisection=section.allow_multiple,
+                                                     clinical_data=data)
+        except KeyError:
```

Comment:
What scenario where you seeing a KeyError? I can't reproduce

Reply:
I forgot to make the rest of the change (which was removing the loading of the clinical data in the dashboard_view, and transferring the responsibility to `get_form_value` which already defines the logic for it.